todocli/test/test_datetime_parser.py

```diff
@@ -0,0 +1,106 @@
+import unittest
+from datetime import datetime, timedelta
+
+from todocli.datetime_parser import (
+    parse_datetime,
+    add_day_if_datetime_is_in_past,
+    ErrorParsingTime,
+    TimeExpressionNotRecognized,
+)
+
+
+class TestDatetimeParser(unittest.TestCase):
+    def test_sadd_day_if_datetime_is_in_past(self):
+        dt_now = datetime.now()
+        dt = dt_now - timedelta(minutes=1)
+        dt = add_day_if_datetime_is_in_past(dt)
+        assert dt.day > dt_now.day
+
+        dt = dt_now + timedelta(hours=1)
+        dt = add_day_if_datetime_is_in_past(dt)
+        assert dt.day == dt_now.day
+
+    def test_american_time1(self):
+        input_str = "07:00 pm"
```

Comment:
Oh wait. Does American time mean using 'am/pm'? Or the order of month and day?

Reply:
Yes, I may not have been very clear about what I mean by "American" time.
I do indeed mean the use of AM / PM. I also mean the Month/Day(/Year) notation.

So the changes introduced in this PR allow the use of "AM/PM" and the input of the date values in the format MM/DD.

Reply:
I see. I never knew AM/PM as an American thing. Very interesting! :) 

Still, lets remove all mentions of 'american' in functions and stuff, since its hard to tell what that means. These tests could be called `test_am_pm` or something more explicit.

Reply:
Yes, I agree, that naming was kind of inappropriate.
I removed all references to "american" in the PR. I renamed all relevant functions to ..._am_pm_...
This should resolve this conversation.